products/protractor.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+
+releaseDateColumn: true
+activeSupportColumn: Community Support
+extendedSupportColumn: Commercial Support
+eolColumn: false
```

Comment:
```suggestion
eolColumn: Community Support
extendedSupportColumn: Commercial Support
```

When only a single column must be shown, we prefer using the `eolColumn` instead of the `activeSupportColumn`. The `activeSupportColumn` is used only when there are two support phases (like for example for https://endoflife.date/angular).

Reply:
Understood. Done